k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -530,6 +530,7 @@ public void onCreate() {
 
         super.onCreate();
         app = this;
+        DI.start(this);
```

Comment:
is `DI` an established name for this class? seems unnecessarily nondescriptive

Reply:
Probably not. I wanted it to be short so Java code can call `DI.get()` instead of `DependencyInjection.get()` or something like that. Do you have a name suggestion?

Reply:
Hm, yeah I see how a longer name can get annoying fast. I'm okay with this.